webui/react/Makefile

```diff
@@ -6,6 +6,7 @@ get-deps:
 	# WARN this module also depends on the swagger generated api client
 	# which is not built or updated here. To set up this dependency checkout
 	# the swagger generated api client module at ../api-ts-sdk
+	cp -r ../api-ts-sdk/src src/services/api-ts-sdk
```

Comment:
question: should we check to make sure `../api-ts-sdk/src` exists before running this command?

Reply:
hmmm we could add something but already it will fail with proper message and we have a warning about this dependency in the make file
```
(determined) hrz ➜ determined git:(master) ✗ cp ~/asdfasdf /tmp                                                                                                                                                                                                                                                                                                          
cp: cannot stat '/home/hmd/asdfasdf': No such file or directory
```